src/sql/workbench/contrib/notebook/test/browser/dataResourceDataProvider.test.ts

```diff
@@ -47,67 +48,69 @@ export class TestSerializationProvider implements azdata.SerializationProvider {
-	let tempFolderPath: string;
 	let fileDialogService: TypeMoq.Mock<TestFileDialogService>;
 	let serializer: ResultSerializer;
-	let dataResourceDataProvider: DataResourceDataProvider;
+	let notificationService: TestNotificationService;
+	let serializationService: SerializationService;
+	let instantiationService: TypeMoq.Mock<InstantiationService>;
+	let cellModel = TypeMoq.Mock.ofType(CellModel);
+
+	// Create test data with two rows and two columns
+	let source: IDataResource = {
+		data: [{ 0: '1', 1: '2' }, { 0: '3', 1: '4' }],
+		schema: { fields: [{ name: 'col1' }, { name: 'col2' }] }
+	};
+	let resultSet: ResultSetSummary = {
+		batchId: 0,
+		columnInfo: [{ columnName: 'col1' }, { columnName: 'col2' }],
+		complete: true,
+		id: 0,
+		rowCount: 2
+	};
 
 	suiteSetup(async () => {
-		// Create test data with two rows and two columns
-		let source: IDataResource = {
-			data: [{ 0: '1', 1: '2' }, { 0: '3', 1: '4' }],
-			schema: { fields: [{ name: 'col1' }, { name: 'col2' }] }
-		};
-		let resultSet: ResultSetSummary = {
-			batchId: 0,
-			columnInfo: [{ columnName: 'col1' }, { columnName: 'col2' }],
-			complete: true,
-			id: 0,
-			rowCount: 2
-		};
-		let cellModel = TypeMoq.Mock.ofType(CellModel);
 		let notebookModel = await createandLoadNotebookModel();
 		cellModel.setup(x => x.notebookModel).returns(() => notebookModel);
-		tempFolderPath = path.join(os.tmpdir(), `TestDataResourceDataProvider_${uuid.v4()}`);
-		await fs.mkdir(tempFolderPath);
 
 		// Mock services
 		let editorService = TypeMoq.Mock.ofType(TestEditorService, TypeMoq.MockBehavior.Strict);
 		editorService.setup(x => x.openEditor(TypeMoq.It.isAny())).returns(() => Promise.resolve(undefined));
 		let contextService = new TestContextService();
 		fileDialogService = TypeMoq.Mock.ofType(TestFileDialogService, TypeMoq.MockBehavior.Strict);
-		let _notificationService = new TestNotificationService();
-		let _serializationService = new SerializationService(undefined, undefined); //_connectionService _capabilitiesService
-		_serializationService.registerProvider('testProviderId', new TestSerializationProvider());
+		notificationService = new TestNotificationService();
+		serializationService = new SerializationService(undefined, undefined); //_connectionService _capabilitiesService
+		serializationService.registerProvider('testProviderId', new TestSerializationProvider());
 		serializer = new ResultSerializer(
 			undefined, // IQueryManagementService
 			undefined, // IConfigurationService
 			editorService.object,
 			contextService,
 			fileDialogService.object,
-			_notificationService,
+			notificationService,
 			undefined // IOpenerService
 		);
-		let _instantiationService = TypeMoq.Mock.ofType(InstantiationService, TypeMoq.MockBehavior.Strict);
-		_instantiationService.setup(x => x.createInstance(TypeMoq.It.isValue(ResultSerializer)))
+		instantiationService = TypeMoq.Mock.ofType(InstantiationService, TypeMoq.MockBehavior.Strict);
```

Comment:
FYI there's a TestInstantiationService that provides the ability to stub out services - little easier to use than using TypeMoq

https://github.com/microsoft/azuredatastudio/blob/main/src/vs/platform/instantiation/test/common/instantiationServiceMock.ts#L18